plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -338,6 +339,24 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'xAxisLabelRotation',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              clearable: false,
+              label: t('Rotate x axis label'),
+              choices: [
+                [0, '0°'],
+                [45, '45°'],
```

Comment:
Maybe add `90`, too?

Reply:
No need for 90 after discussion with jun

Reply:
That's fine, I'm not a fan of 90 degree rotation, either.

But I just wanna point out that it is actually used in the official examples of [HighCharts](https://api.highcharts.com/highcharts/xAxis.labels.rotation) and [AMCharts](https://www.amcharts.com/demos/column-with-rotated-series/), so I'd imagine some users may actually prefer it for some cases.

Reply:
![tenor](https://user-images.githubusercontent.com/67837651/106987501-75553700-6722-11eb-942a-b17853b46548.gif)

Reply:
No they won't. ECharts API does not allow that. [ECharts options only allow -90 to 90.](https://echarts.apache.org/en/option.html#xAxis.axisLabel.rotate)

Reply:
We should probably add a custom validator for this, btw. Not super important, though.